transport/tchannel/handler_test.go

```diff
@@ -663,3 +663,32 @@ func TestGetSystemError(t *testing.T) {
 		})
 	}
 }
+
+func TestIsTChannelTimeoutError(t *testing.T) {
+	tests := []struct {
+		giveErr  error
+		wantResult bool
+	}{
+		{
+			giveErr:  errors.New("test"),
+			wantResult: false,
+		},
+		{
+			giveErr:  tchannel.NewSystemError(tchannel.ErrCodeBusy, "test"),
+			wantResult: false,
+		},
+		{
+			giveErr:  yarpcerrors.Newf(_tchannelCodeToCode[tchannel.ErrCodeTimeout], "time out"),
+			wantResult: true,
+		},
+		{
+			giveErr:  tchannel.NewSystemError(tchannel.ErrCodeTimeout, "time out"),
+			wantResult: true,
+		},
+	}
+	for i, tt := range tests {
+		t.Run(string(i), func(t *testing.T) {
+			assert.Equal(t, tt.wantResult, isTChannelTimeoutError(tt.giveErr))
```

Comment:
Let's add an error message to the `assert`, this would gives us a better indication of what, if anything, went wrong.
```suggestion
			assert.Equal(t, tt.wantResult, isTChannelTimeoutError(tt.giveErr), "unexpected result, expected %t, got %t", tt.wantResult, isTChannelTimeoutError(tt.giveErr))
```

Reply:
Sure, added :)